tasks/gulp/backstop-config.js

```diff
@@ -83,10 +85,10 @@ module.exports = ({ host, port, components }) => ({
   report: ['browser'],
   engine: 'playwright',
   engineOptions: {
-    args: ['--no-sandbox'],
+    args: ['--no-sandbox', '--allow-file-access-from-files', '--disable-web-security'],
```

Comment:
stops there being a cors error when we try to load stuff from css (like the svg in the footer and fonts)